### ISSUE TITLE
Cannot configure project with release 2.5.0/2.5.1

### ISSUE THREAD

**Issue by 3flex** [BreadMoirai/github-release-gradle-plugin#64]
Error:

```
* What went wrong:
An exception occurred applying plugin request [id: 'com.github.breadmoirai.github-release']
> Failed to apply plugin 'com.github.breadmoirai.github-release'.
   > Could not create an instance of type com.github.breadmoirai.githubreleaseplugin.GithubReleaseExtension.
      > No signature of method: org.gradle.api.internal.provider.DefaultProperty.call() is applicable for argument types: (com.github.breadmoirai.githubreleaseplugin.GithubReleaseExtension$_closure1) values: [com.github.breadmoirai.githubreleaseplugin.GithubReleaseExtension$_closure1@[27](https://github.com/detekt/detekt/actions/runs/6493009537/job/17633076172?pr=6542#step:5:28)036385]
        Possible solutions: wait(), any(), wait(long), map(org.gradle.api.Transformer), map(org.gradle.api.Transformer), value(java.lang.Object)
```

I'm not sure why but using closures to set the default values in the extension seems to be causing the problem - even though that's how they were applied in previous versions.

**Comment by 3flex** [comment]
@BreadMoirai this is a significant issue blocking upgrades - I've submitted a PR that fixes it, can this be merged and a new release cut please?

I'm very keen to start using v2.5+ that includes a fix for #58 as easier versions cause a lot of issues in the IDE.

**Comment by simonpoole** [comment]
I have a project that is facing the same issue, confounded by the fact that the ast dependency that 2.4.1 has seems to no longer be available, which breaks all builds on systems that don't have that cached.
